test/filters.js

```diff
@@ -184,6 +194,14 @@ describe('filters', function () {
       () => test('{{ 183.357 | minus: 12 }}', '171.357'))
     it('should convert first arg as number', () => test('{{ "4" | minus: 1 }}', '3'))
     it('should convert both args as number', () => test('{{ "4" | minus: "1" }}', '3'))
+    it('should return {"type":"days","value":730}', () => {
+      try {
+        const dst = {type: "days", value: 730};
+        return test('{{ from_date | minus: to_date}}', JSON.stringify(dst))
```

Comment:
since we expect this to be used with `assign` it will be nice if we can add a test for that?